drizzle-orm/src/sqlite-core/README.md

```diff
@@ -247,6 +247,7 @@ Declaring indexes, foreign keys and composite primary keys
 
 ```typescript
 import { sqliteTable, foreignKey, primaryKey, text, integer, index, uniqueIndex } from "drizzle-orm/sqlite-core";
+import type { AnySQLiteColumn } from "drizzle-orm/sqlite-core";
```

Comment:
Could you move it to the import above? I know we use type imports in the ORM codebase, but for examples it's simpler to do all the imports together.

Reply:
Sure thing